packages/camera/camera_avfoundation/ios/Classes/CameraPlugin.m

```diff
@@ -299,6 +299,14 @@ - (void)handleCreateMethodCall:(FlutterMethodCall *)call
   });
 }
 
+// Returns integer value if provided and positive, or nil if not specified or negative.
+NSNumber *toPositiveOrNull(NSNumber *number) {
+  int value = (!number || [number isEqual:[NSNull null]] || isnan([number doubleValue]))
```

Comment:
> `[number isEqual:[NSNull null]]`

An `NSNumber` cannot be `NSNull`, because they are unrelated classes. If it's possible to pass `NSNull` to this method, then `NSNumber` is not the correct type for the parameter.

Reply:
`number` type is `id` now in function declaration.